controllers/feeds/getFeedsV2.js

```diff
@@ -17,7 +17,7 @@ const {
   modifyAnonimityPost,
   isPostBlocked,
   modifyReactionsPost,
-  modifyFeedIsFollowingTarget
+  isUserFollowAuthor
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `isUserFollowAuthor` is imported but the old function `modifyFeedIsFollowingTarget` is not removed. Make sure to remove unused imports to keep the code clean and maintainable.
```diff
-  modifyFeedIsFollowingTarget
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->